test/NuGet.Clients.Tests/NuGet.VisualStudio.Implementation.Test/Extensibility/VsPathContextProviderTests.cs

```diff
@@ -439,6 +439,42 @@ public void CreateSolutionContext_WithConfiguredFallbackPackageFolders()
                 actual.FallbackPackageFolders.Cast<string>().ToArray());
         }
 
+        [Fact]
+        public void CreateSolutionContext_WithPackagesConfig()
```

Comment:
Can you add another test for the new `ryCreateSolutionContext(string solutionDirectory, out IVsPathContext2 context)` API added?

Reply:
sure, will add it